selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumWebDriverRelatedModule.java

```diff
@@ -45,4 +49,12 @@ public Entrance getEntrance(
       return new CookieEntrance(seleniumWebDriver);
     }
   }
+
+  @Provides
```

Comment:
Could you, please, explain, why did you place this provider here?

Reply:
Because in che we return CheTestTestOrganizationServiceClientImpl and in the codenvy we return other implementation of TestOrganizationServiceClient interface

Reply:
I see. But the propose of **CheSeleniumWebDriverRelatedModule** is the classes which are injecting webdriver itself.